spec.bs

```diff
@@ -2737,11 +2782,15 @@ algorithms to achieve the outcomes described in the above explanatory content.
      creating a permissions policy$] given |navigable|'s [=fenced navigable container=], |origin|,
      and <var ignore>fenced</var> set to true.
 
-     Note: This is identical to the [=Document/permissions policy=] that will be created when the
-     navigation constructs the ultimate {{Document}} for this pending navigation. We create it now
-     and run tests on it since this is the appropriate time to determine if a navigation will fail,
-     and then throw it away. If the navigation succeeds, it will be recreated identically and
-     unconditionally installed on the {{Document}}.
+     Note: This is almost identical to the [=Document/permissions policy=] that will be created when
```

Comment:
nit: I'd link "created" to the "derive permissions policy directly from fenced frame config instance" and avoid the parenthesis "(with the [=foo=] algorithm)". I think this would be more clear.